docs/running-rules.md

```diff
@@ -38,12 +38,12 @@ Explore the Semgrep Registry by following these steps:
 1. See the [Semgrep Registry](https://semgrep.dev/r) and choose a rule.
 2. On the page of the rule, click **Run Locally**.
 3. Copy the code for local install, and then add the path to the source code you want to check in your terminal:
-    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" [<span className="placeholder">PATH/TO/SRC</span>]</code></pre>
+    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
 4. Optional: Run registry rules simultaneously with local rules:
    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/MYRULE.YAML PATH/TO/SRC</span></code></pre>
 
-:::info
-Try Semgrep [Rule Board](semgrep-app/rule-board.md) to determine which rules Semgrep uses and what action Semgrep undertakes when it generates a finding.
+:::tip
+See Semgrep [Rule Board](semgrep-app/rule-board.md) to select which rules Semgrep uses and what action it undertakes when it generates a finding.
```

Comment:
Hmm... okay so for running-rules LOCALLY, the implication is we're using Semgrep CLI.

It is technically correct that we can run Semgrep APP rule board locally, but we haven't explained how to connect the LOCAL set up to the APP (webapp) setup.I'd suggest removing this entirely, it's not really related and gives the user the idea that the Rule Board and Registry are somehow related without explaining what the relation is.

Reply:
This is true, I have the same feeling @s-santillan. Will remove this.